feat(apps-ui-kit): update error message styles in Input, Select, and TextArea components

Juligs opened the pull request:
fixes #7415

Juligs committed:
refactor(apps-ui-kit): modify ui-kit config to allow multitheme (#7309)

fixes #7300

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
refactor(apps-ui-kit): modify ui-kit config to allow multitheme (#7309)

fixes #7300

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
feat(apps-ui-kit): add iota-names color palette and typography (#7331)

fixes #7302

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
feat(apps-ui-kit): add iota-names color palette and typography (#7331)

fixes #7302

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
feat(apps-ui-kit): add iota-names color palette and typography (#7331)

fixes #7302

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
refactor(apps-ui-kit):  update atoms to apply new palettes (Badge-LabelText) (#7355)

fixes #7344

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
refactor(apps-ui-kit):  update atoms to apply new palettes (Badge-LabelText) (#7355)

fixes #7344

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
refactor(apps-ui-kit):  update atoms to apply new palettes (Badge-LabelText) (#7355)

fixes #7344

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

evavirseda committed:
feat(apps-ui-kit): update atoms to apply new palettes (`LisItem` - `VisualAssetCard`) (#7349)

# Description of change

Please write a summary of your changes and why you made them.

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7345

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update atoms to apply new palettes (`LisItem` - `VisualAssetCard`) (#7349)

# Description of change

Please write a summary of your changes and why you made them.

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7345

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update molecules to apply new palettes (#7354)

# Description of change

nested PR :warning: https://github.com/iotaledger/iota/pull/7349

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7351

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update molecules to apply new palettes (#7354)

# Description of change

nested PR :warning: https://github.com/iotaledger/iota/pull/7349

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7351

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update molecules to apply new palettes (#7354)

# Description of change

nested PR :warning: https://github.com/iotaledger/iota/pull/7349

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7351

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update organisms to apply new palettes (#7359)

# Description of change

nested PR :warning: https://github.com/iotaledger/iota/pull/7354

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7352

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update organisms to apply new palettes (#7359)

# Description of change

nested PR :warning: https://github.com/iotaledger/iota/pull/7354

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7352

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update organisms to apply new palettes (#7359)

# Description of change

nested PR :warning: https://github.com/iotaledger/iota/pull/7354

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7352

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
bricg back border rounded

evavirseda committed:
fix neutral badge border

evavirseda committed:
fix class

evavirseda committed:
fix keyvalue icon color

evavirseda committed:
fix class

evavirseda committed:
fix outline border

evavirseda committed:
filter themes to not show names

evavirseda committed:
fix class

evavirseda committed:
fix tooltip in the wallet

Juligs committed:
feat(apps-ui-kit): update storybook after multi-theme (#7377)

fixes #7374

Juligs committed:
feat(apps-ui-kit): update storybook after multi-theme (#7377)

fixes #7374

Juligs committed:
feat(apps-ui-kit): update storybook after multi-theme (#7377)

fixes #7374

evavirseda committed:
Merge branch 'tooling/make-ui-kit-multi-theme-compatible' into tooling-uikit/fix-missing-styles

evavirseda committed:
fix fonts

Juligs committed:
fix(apps-ui-kit): adjust styles for disabled state and improve label

evavirseda committed:
fix navbar item classes

Juligs committed:
refactor(apps-ui-kit): fix search component styles, add divider color variables, and update Storybook backgrounds

Juligs committed:
feat(apps-ui-kit): update search and table cell styles with new color variables

Juligs committed:
feat(apps-ui-kit): update table header cell, title, and accordion variables

Juligs committed:
fix(apps-ui-kit): correct input border color classes and improve focus/error state handling

Juligs committed:
chore(apps-ui-kit): fix prettier format

Juligs committed:
Revert "feat(apps-ui-kit): update table header cell, title, and accordion variables"

This reverts commit ae15f0b2e9704d2a244a629dc0763f3e9a8be212.

Juligs committed:
feat(apps-ui-kit): update styles for table header cell, title icon, and accordion border

Juligs committed:
feat(apps-ui-kit): update SecondaryText component

evavirseda committed:
Merge branch 'tooling/make-ui-kit-multi-theme-compatible' into tooling-uikit/fix-missing-styles

evavirseda committed:
fix missing conflict

Juligs committed:
Merge branch 'tooling-uikit/fix-missing-styles' into tooling-uikit/update-input-styles

Juligs committed:
chore(apps-ui-kit): update secondary error bg in dark root

evavirseda commented on apps/ui-kit/src/lib/styles/index.css:
```diff
@@ -121,7 +121,8 @@
     --secondary-text-color: theme('colors.iota-neutral.40');
-    --secondary-text-error-color: theme('colors.iota-error.30');
+    --secondary-text-error-color: theme('colors.iota-error.20');
+    --secondary-error-bg-color: theme('colors.iota-error.90');
```
`--secondary-error-bg-color` is not a necessary variable, you can use `--error-surface` for backgrounds, as in other components :pray: 
```
light: --error-surface: theme('colors.iota-error.90');
dark: --error-surface: theme('colors.iota-error.10');
names: --error-surface: theme('colors.names-error.20');
```

evavirseda committed:
feat(apps-ui-kit): review apps and fix missing styles (#7376)

# Description of change

Please write a summary of your changes and why you made them.

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7375

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

---------

Co-authored-by: Juliana <115430927+Juligs@users.noreply.github.com>
Co-authored-by: Juligs <julicodes29@gmail.com>

evavirseda committed:
refactor(apps-ui-kit): modify ui-kit config to allow multitheme (#7309)

fixes #7300

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

evavirseda committed:
feat(apps-ui-kit): update atoms to apply new palettes (`LisItem` - `VisualAssetCard`) (#7349)

Please write a summary of your changes and why you made them.

fixes https://github.com/iotaledger/iota/issues/7345

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

Juligs committed:
Merge branch 'tooling/make-ui-kit-multi-theme-compatible' into tooling-uikit/update-input-styles

Juligs committed:
chore(apps-ui-kit): remove unused secondary error bg

Juligs committed:
Merge branch 'tooling/make-ui-kit-multi-theme-compatible' into tooling-uikit/update-input-styles

Juligs committed:
chore: revert unnecessary changes

Juligs committed:
chore: revert unnecessary changes without prettier

evavirseda committed:
fix fmt

evavirseda committed:
Merge branch 'tooling/make-ui-kit-multi-theme-compatible' into tooling-uikit/update-input-styles

evavirseda committed:
fix lockfile

VmMad commented on apps/ui-kit/src/lib/components/atoms/secondary-text/SecondaryText.tsx:
only apply the class when necessary

Juligs committed:
chore(apps-ui-kit): enhance layout

evavirseda committed:
Merge branch 'tooling/make-ui-kit-multi-theme-compatible' into tooling-uikit/update-input-styles
